bigtable-hbase-integration-tests/src/test/java/com/google/cloud/bigtable/hbase/AbstractTest.java

```diff
@@ -30,6 +30,8 @@
 public abstract class AbstractTest {
   protected DataGenerationHelper dataHelper = new DataGenerationHelper();
 
+  protected Logger logger = new Logger(this.getClass());
```

Comment:
We can't use "this" with static.  "this" is needed to get a logger that will print out the classname.